en/mulesoft/modules/ROOT/pages/ms_composer_rpa_reference.adoc

```diff
@@ -21,15 +21,21 @@ Automate the processing of customer refund requests to make them quicker.
 * Customer information updates +
 Update customer information automatically across different systems that do not have APIs and that are accessible only through the UI.
 
-== Invoke Two Composer Flows for RPA
+== Invoke Short-Running RPA Processes
+
+When invoking short-running RPA processes from Composer, invoke a single flow by choosing the *Use response in this flow* button. The RPA execution is retrieved by the RPA step and subsequent steps can use the RPA step datapills.
+
+== Invoke Long-Running RPA Processes
 
 When invoking RPA processes from Composer, a two-flow Composer scenario is only required when the RPA process takes longer than the allowed completion timeout period, which is three minutes.
 
-For long-running processes, you can configure two flows that finish within the Composer flow timeout period, which avoids blocking other events that would be queued for processing.
+For long-running processes, you can configure two flows that finish within the Composer flow timeout period, which avoids blocking other events that would be queued for processing while waiting for the RPA process to complete.
```

Comment:
```suggestion
For long-running processes, you can configure two flows that finish within the Composer flow timeout period to avoid blocking other in-queue events from processing while waiting for the RPA process to complete.
```